modules/smarthome/ratiotherm/off.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/python3
+import sys
+import os
+import time
+
+named_tuple = time.localtime()  # getstruct_time
+time_string = time.strftime("%m/%d/%Y, %H:%M:%S ratiotherm off.py", named_tuple)
```

Comment:
Warum hast Du hier nicht das Python-Logging Modul verwendet?

Reply:
Es wurde das für ratiotherm das python logging modul verwendet.
Sofern die Anwendung okay ist, kann ich die anderen Geräte auch umstellen.
Dafür habe ich ein zentrales Smartutils.py geschrieben, welches dann in allen Smarthomegeräten angewendet wird. Da der PYTHONPATH bereits auf ..openWB/packages/ verweist, habe ich die smartutils.py im Verzeichnis  ..openWB/packages/modules/common abgelegt so dass es im gesamten .../Smarthome/... Folder importiert werden kann

Reply:
Den Pfad finde ich nicht richtig, da es so zum Vermischen zweier Quellen kommt (offizielle Module und SmartHome). Da der Ordner `packages` so auch nach 2.0 gemerged wird, kann es leicht zu einem Durcheinander kommen.

Reply:
wie abgesprochen wird neu das ...packages\smarthome\ Verzeichnis für gemeinsame import files verwendet.